### PR TITLE
More Rank Information

### LINKED ISSUE #3294: More info for ranks
Posted in discord
drlgzqcu8aesk_z

My idea to achieve this without a wall of text would be to make another circle, around progress through current rank that shows progress towards reset.

//cc @bhollis @inexorableAce

### PR DESCRIPTION
**Pull request by delphiactual**
Closes #3294 

screenshot from 2018-11-09 21-52-37

~~Not sure what is happening with the circle for Glory any help would be appreciated~~

### CODE REVIEW

**Comment by bhollis** [comment]
Neat!

What do you think of making the outer ring a different (more subtle) color? One thing I'm not sure of is how to indicate that it's the full-reset amount.

File: src/app/progress/CrucibleRank.tsx [hunk 1]
```diff
@@ -28,6 +32,9 @@ export function CrucibleRank(props: CrucibleRankProps) {
         <div className="faction-level">
           {progress.progressToNextLevel} / {progress.nextLevelAt}
         </div>
+        <div className="faction-level">
+          {progress.currentProgress} / {rankTotal}
```
Comment (bhollis) [inline src/app/progress/CrucibleRank.tsx]: Maybe we should just display this as "% to reset"?

**Comment by delphiactual** [comment]
screenshot from 2018-11-10 13-39-12

**Comment by bhollis** [comment]
Awesome. Can we take the percentage to the nearest integer?

**Comment by kyleshay** [comment]
Like the change but the percentage by itself is sort of confusing. Could we add some additional text (or some hover text) that explains what that percentage is?

**Comment by kyleshay** [comment]
Maybe mouse over on the rings themselves for some more verbose information, but otherwise something like `66% to prestige`?

**Comment by bhollis** [comment]
Yeah a little help text would be great. We can look up the localized name of the last step, or just make a string like `X% to reset`.

**Pull request merged**